web/src/people.ts

```diff
@@ -14,15 +14,66 @@ import * as settings_config from "./settings_config";
 import * as settings_data from "./settings_data";
 import * as util from "./util";
 
-let people_dict;
-let people_by_name_dict;
-let people_by_user_id_dict;
-let active_user_dict;
-let non_active_user_dict;
-let cross_realm_dict;
-let pm_recipient_count_dict;
-let duplicate_full_name_data;
-let my_user_id;
+export type ProfileData = {
+    value: string;
+    rendered_value: string;
```

Comment:
`rendered_value` is optional.
https://github.com/zulip/zulip/blob/b5e572811257a33a25494af34a5231823cbe5f02/zerver/openapi/zulip.yaml#L562-L570